Formula/lima.rb

```diff
@@ -0,0 +1,25 @@
+class Lima < Formula
+  desc "Linux virtual machines"
+  homepage "https://github.com/AkihiroSuda/lima"
+  url "https://github.com/AkihiroSuda/lima/archive/v0.3.0.tar.gz"
+  sha256 "0debed5cd2fcb34ce90114c4f07a3610a3b2d4aaf4592608135e9053cb5ce213"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "qemu"
```

Comment:
Slightly off-topic here, but the qemu formula is currently broken until https://github.com/Homebrew/homebrew-core/pull/79479 gets merged.

Reply:
The qemu part was merged can you check the pr again please @carlocab?